experiments/integrated/ozark/ozark_simulation.jl

```diff
@@ -1,8 +1,8 @@
 t0 = FT(120 * 3600 * 24)# start mid year
 N_days = 120
 tf = t0 + FT(3600 * 24 * N_days)
-dt = FT(60)
-n = 60
+dt = FT(240)
+n = 15
```

Comment:
Maybe we can add a comment to say what `n` is, for clarity
```
# save output at every n * dt
```
we see it in the next line, but it's easier to read line by line?

Reply:
will do, thank you!